relay-event-normalization/src/normalize/span/tag_extraction.rs

```diff
@@ -27,130 +27,6 @@ use crate::utils::{
     extract_transaction_op, http_status_code_from_span, MAIN_THREAD_NAME, MOBILE_SDKS,
 };
 
-/// A list of supported span tags for tag extraction.
-#[derive(Clone, Debug, PartialEq, Eq, PartialOrd, Ord)]
-#[allow(missing_docs)]
-pub enum SpanTagKey {
```

Comment:
Instead of storing a `BTreeMap<SpanTagKey, Annotated<String>>`, we now have a `SentryTags` struct.